templates/nodegroupconfiguration-redos.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: deckhouse.io/v1alpha1
+kind: NodeGroupConfiguration
+metadata:
+  name: redos-selinux-packages
+  {{- include "helm_lib_module_labels" (list .) | nindent 2 }}
+spec:
+  bundles:
+  - 'redos'
+  weight: 100
+  nodeGroups:
+   - '*'
+  content: |
+    # Copyright 2024 Flant JSC
+    #
+    # Licensed under the Apache License, Version 2.0 (the "License");
+    # you may not use this file except in compliance with the License.
+    # You may obtain a copy of the License at
+    #
+    #     http://www.apache.org/licenses/LICENSE-2.0
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
+
+    # bb-event-on 'bb-package-installed' 'post-install'
+    # post-install() {
+    #   bb-log-info "Setting reboot flag due to selinux-policy was installed"
+    #   bb-flag-set reboot
+    # }
```

Comment:
Why we need to keep these commented lines?

Reply:
We don't. Fixed